pset-iterators.js

```diff
@@ -115,6 +135,13 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  return president.took_office.split('-')[0] >= 1850 && president.took_office.split('-')[0] <= 1900
```

Comment:
@kcycle nice job, to be on the safe side, you could have also converted the took_office year to an integer to avoid any issues related to comparing a string to an integer...but I checked and it still works out, looks like less than (`<`) and greater than (`>`) don't do a strict comparison